casper/src/main/resources/RevVault.rho

```diff
@@ -102,7 +104,7 @@ in {
         contract _newVault(@ownRevAddress, purse, ret) = {
           new revVault in {
 
-            ret!(bundle+{*revVault}) |
+            ret!(("Right", bundle+{*revVault})) |
```

Comment:
when does this _newVault contract return a "Left" value?

Reply:
This changes in the next commit, that was a bandaid so that I could have the tests pass without changing everything at once.